endpoints/context_dev.go

```diff
@@ -123,9 +110,8 @@ func (c *tokeninfoContext) CurrentOAuthUser(scope string) (*user.User, error) {
 	return &user.User{Email: ti.Email}, nil
 }
 
-// tokeninfoContextFactory creates a new tokeninfoContext from r.
-// To be used as auth.go/ContextFactory.
-func tokeninfoContextFactory(r *http.Request) Context {
-	ac := appengine.NewContext(r)
-	return &tokeninfoContext{ac}
+// tokeninfoAuthenticatorFactory creates a new tokeninfoAuthenticator from r.
+// To be used as auth.go/AuthenticatorFactory.
+func tokeninfoAuthenticatorFactory() Authenticator {
```

Comment:
The `AuthenticatorFactory` needs to be a function so that an authenticator can be created per-context.  This is a function to fulfill the "interface" of that variable.